htdocs/core/class/commoninvoice.class.php

```diff
@@ -1563,13 +1456,13 @@ public function makeStripeSepaRequest($fuser, $did = 0, $type = 'direct-debit',
 					$idtransferfile = 0;
 
 					// Update the direct debit payment request of the processed invoice to save the id of the prelevement_bon
-					$sql = "UPDATE".MAIN_DB_PREFIX."prelevement_demande SET";
+					$sql = "UPDATE ".MAIN_DB_PREFIX."prelevement_demande SET";
```

Comment:
You must introduce the creation of processing request 4 lines before this one at the // TODO Create a prelevement_bon...